app/views/clients/rollup/_assessments.html.haml

```diff
@@ -84,6 +94,40 @@
               %td= HudUtility2024.prioritization_status assessment.PrioritizationStatus
               %td= assessment.user&.name
               %td.zero-width
+  - if custom_hmis_assessments.any?
+    %table.table
+      %thead
+        %tr
+          %th Assessment Name
+          %th Project
+          %th Collection Date
+          %th Staff
+          %th View in HMIS
+          %th
+      %tbody
+        - custom_hmis_assessments.each_with_index do |(name, (assessment, *assessments)), i|
+          - any_buttons = assessments.any?
+          - cz = "assessment_type_#{i}"
+          %tr{ class: ( 'assessment__new-type' if any_buttons )}
+            %td= link_to name, hmis_client_client_assessment_path(@client, assessment), class: 'btn btn-primary btn-muted btn-xs', data: { loads_in_pjax_modal: true }
+            %td= assessment.enrollment.project.warehouse_project.name(current_user)
+            %td= assessment.AssessmentDate
+            %td= assessment.user&.name
+            - path = "client/#{assessment.enrollment.client.id}/enrollments/#{assessment.enrollment.id}/assessments/#{assessment.id}"
+            %td= link_to 'HMIS', url_for_hmis(assessment.enrollment.data_source, path), target: :_blank
```

Comment:
Can we add `Hmis::Hud::CustomAssessment` to this function instead? 
https://github.com/greenriver/hmis-warehouse/blob/f7d4269c21328c88f490986cd89285898805b422/app/models/grda_warehouse/data_source.rb#L556-L576

Reply:
Yes, much better thanks!.